modules/ingest-common/src/test/resources/rest-api-spec/test/ingest/120_grok.yml

```diff
@@ -158,6 +158,6 @@ teardown:
 ---
 "Test Grok Patterns Retrieval":
   - do:
-      ingest.processor.grok: {}
+      ingest.processor_grok: {}
```

Comment:
wondering if you were the first one ever running these tests...

Reply:
I think I don't follow this change, why the underscore instead of the dot? the API is called `ingest.processor.grok` in the spec right? This should fail the build I think.

Reply:
I renamed `ingest.processor.grok` to `ingest.processor_grok` in the spec too, so this should work.  The reason for the underscore instead of the dot is that an underscore returns a new namespace in the REST clients, but we have only one thing in this namespace, so makes more sense just to have it at the same level as the other ingest methods.

Reply:
Yes, I see that now. And the name of the api didn't match the name of the file. All good. I am going to open a PR to validate that the name of the api matches the filename, I've seen enough of these errors.